.travis.yml

```diff
@@ -14,9 +14,11 @@ env:
 
 android:
   components:
+    #tools required
```

Comment:
space after `#` and before text, seems to be the standard, please use here and below